data/alpha/register/internal-drainage-board.yaml

```diff
@@ -0,0 +1,10 @@
+fields:
+- internal-drainage-board
+- name
+- legislation
+- start-date
+- end-date
+phase: alpha
+register: internal-drainage-board
+registry: department-for-environment-food-rural-affairs
+text: 'A register of internal drainage boards in England. /n /n Internal drainage boards (IDB) are public bodies which manage water levels in some areas where there is a special need for drainage. These areas are known as internal drainage districts. Each internal drainage board has permissive powers to undertake work to provide water level management within their internal drainage district. This includes undertaking works to reduce flood risk to people and property and manage water levels for local needs'
```

Comment:
These should be `\n\n` rather than `/n/n` assuming they are meant to be new lines